### PR TITLE
Added class to set S3 events

### PR DESCRIPTION
**Pull request by abetomo**
Added class to set s3 events.
In the next PR add S3Events to main.js.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
     "test": "test"
   },
   "scripts": {
-    "test": "standard && standard bin/node-lambda && mocha"
+    "lint": "standard && standard bin/node-lambda",
+    "test": "npm run lint && npm run unit",
+    "unit": "mocha"
```
Comment (abetomo) [inline package.json]: It has nothing to do with subject matter, but cleaned the `scripts`.

File: lib/s3_events.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+/**
+ * Do not create Bucket of S3
```
Comment (DeviaVir) [inline lib/s3_events.js]: `Do not create S3 bucket`?

File: lib/s3_events.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  }
+
+  _addPermission (params) {
+    return new Promise((resolve, reject) => {
+      const _params = this._addPermissionParams(params)
+      this.lambda.addPermission(_params, (err, data) => {
+        if (err) {
+          if (err.code !== 'ResourceConflictException') reject(err)
+          // If it exists it will result in an error but there is no problem.
+          resolve('Already exists permission')
```
Comment (DeviaVir) [inline lib/s3_events.js]: `Permission already set`

**Comment by abetomo** [comment]
@DeviaVir Thank you!
Your advice was very helpful.

**Pull request merged**